Plugin/CachePlugin.php

```diff
@@ -21,24 +23,34 @@ class CachePlugin
      */
     private $cacheButton;
 
+    /**
+     * @var AuthorizationInterface
+     */
+    private $_authorization;
+
     /**
      * CachePlugin constructor.
+     *
      * @param CacheButton $cacheButton
+     * @param Context     $context
      */
-    public function __construct(CacheButton $cacheButton)
+    public function __construct(CacheButton $cacheButton, Context $context)
```

Comment:
Please inject AuthorizationInterface directly into this class without using Context class.